src/libraries/Common/src/System/Security/Cryptography/CngPkcs8.cs

```diff
@@ -277,8 +276,7 @@ private static Pkcs8Response ImportPkcs8(AsnWriter pkcs8Writer)
                         }
                         finally
                         {
-                            CryptographicOperations.ZeroMemory(decryptedSpan);
-                            CryptoPool.Return(decrypted.Array!, clearSize: 0);
+                            CryptoPool.Return(decrypted);
```

Comment:
This had been clearing nothing, and now it's clearing the whole span.  Is that intentional?

Reply:
It was being cleared by the (now removed) `ZeroMemory` right above it. `decryptedSpan` is a `Span<byte>` over `decrypted`.

Reply:
Ok, I need more coffee. Thanks.